breeze

```diff
@@ -2211,6 +2259,12 @@ function breeze::flag_local_file_mounting() {
 -l, --skip-mounting-local-sources
         Skips mounting local volume with sources - you get exactly what is in the
         docker image rather than your current local sources of Airflow.
+
+--skip-mounting-files
```

Comment:
Why do we need an option to do this? It seems harmless to just always mount this, and the less options/configuration in breeze the better -- it is already hard to follow everything.

Reply:
We need it badly. The problem is that when you install from wheels AND have sources mounted, python interpreter behaves differently in python 3 is to look for airflow in the current working directory. so if you mount sources AND have packages installed, behaviour depends on which directory is your cwd. This is bad and confusing. 

On the other hand there are cases where you do need the sources  - even if you install the packages, for example when you want to test setup.py behaviour with installed dependencies. It helped me a lot with my dependency work. So we need that option.

Reply:
This is not sources though - it's controlling if `/files` and `/dist` are mounted. They seems safe to do always

Reply:
Ah. I see - in the snipped from GH I saw the sources option displayed in the first line so I thought you were commenting on that. 

There is no particular use case for that I can think of . It was just option I realized I missed when I was moving /dist to the right docker compose file, so I just added it. 

But now that you mentioned it, indeed it can be removed - I checked that we never switch it off in the CI anyway. So yeah. I am happy to remove it together with the MOUNT_FILES env variable.

Reply:
Removed.